Remotion/ObjectBinding/Web.Test/IndividualControlTests/BocDropDownMenuUserControl.ascx

```diff
@@ -29,7 +29,8 @@
   <td colSpan=4><remotion:boctextvalue id=FirstNameField runat="server" ReadOnly="True" datasourcecontrol="CurrentObject" PropertyIdentifier="FirstName"></remotion:boctextvalue>&nbsp;<remotion:boctextvalue id=LastNameField runat="server" ReadOnly="True" datasourcecontrol="CurrentObject" PropertyIdentifier="LastName"></remotion:boctextvalue></td></tr>
 <tr>
   <td></td>
-  <td><remotion:BocDropDownMenu id=PartnerField runat="server" datasourcecontrol="CurrentObject" PropertyIdentifier="Partner" Width="20em" ControlServicePath="BocDropDownMenuWebService.asmx" ControlServiceArguments="ControlServiceArgs" /></td>
+  <%-- style="right:0px;position:absolute;width:100px" --%>
+  <td><remotion:BocDropDownMenu  id=PartnerField runat="server" datasourcecontrol="CurrentObject" PropertyIdentifier="Partner" Width="20em" ControlServicePath="BocDropDownMenuWebService.asmx" ControlServiceArguments="ControlServiceArgs" /></td>
```

Comment:
Are those changes left-over test code?

Reply:
Yes, part of the [!d testing](https://github.com/re-motion/Framework/pull/391/commits/fe628c034ecb57ccbaba94f33128fa5d408c64d7) commit.

Reply:
Thats what I get for looking at the whole diff :p